pkgs/development/compilers/ghc/8.2.2.nix

```diff
@@ -87,7 +87,8 @@ stdenv.mkDerivation rec {
     sha256 = "1z05vkpaj54xdypmaml50hgsdpw29dhbs2r7magx0cm199iw73mv";
   };
 
-  enableParallelBuilding = true;
+  # https://ghc.haskell.org/trac/ghc/ticket/15449
+  enableParallelBuilding = !buildPlatform.isAarch64;
```

Comment:
I think this should be targetPlatform if it is happening during codegen.

Reply:
Thanks for reviewing. Can we get a confirmation on this from @Ericson2314?

Reply:
`buildPlatform` sounds correct since according the ticket the problem happens when _running_ multithreaded Haskell code on aarch64.

Reply:
Ah ok- I read the codegen part and thought it was talking about the target.